bin/prep-release.php

```diff
@@ -65,7 +65,7 @@ function get_files() {
         "src/libmongocrypt-compat/mongocrypt/*.{c,h}",
         "src/libmongocrypt/src/*.{c,h,h.in}",
         "src/libmongocrypt/src/crypto/*.{c,h}",
-        "src/libmongocrypt/src/mlib/*.{c,h}",
+        "src/libmongocrypt/src/mlib/*.h",
```

Comment:
@alcaeus: This line appears to be redundant with the pattern appearing _after_ the subsequent comment. Perhaps this should be removed entirely?

Reply:
Indeed. Removed in #1848